Microsoft.NET.Build.Containers.IntegrationTests/DockerTestsCollection.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace Microsoft.NET.Build.Containers.IntegrationTests;
+
+[CollectionDefinition("Docker tests")]
+#pragma warning disable CA1711 // Identifiers should not have incorrect suffix
+public class DockerTestsCollection : ICollectionFixture<DockerTestsFixture>
```

Comment:
I'm using `ICollectionFixture` to initialize docker now.
If beneficial, we can move to `AssemblyFixture`; however it is not part of xunit package, and has to be added additionally.

Reply:
Very happy to make the registry more localized - I've got some test scenarios coming up that need a registry configured a different way entirely, and being able to scope that changes to specific tests via a collection will make the entire process a lot easier.